packages/engine/src/worker/mod.rs

```diff
@@ -93,10 +101,12 @@ impl WorkerController {
         })
     }
 
-    /// ### Run the main loop of the Worker
+    /// Runs the main loop of the Worker and [`await`]s for completition.
+    ///
+    /// The main loop allows the worker to receive/register tasks, drive tasks to completion and
```

Comment:
```suggestion
    /// Runs the main loop of the Worker and [`await`]s for completion.
    ///
    /// The main loop allows the worker to receive/register tasks, drive tasks to completion and
```

I'm not necessarily against us calling it a main loop, I don't think it's a bad classification or anything. I just think it might lead to confusion with the simulation main loop. @litvand what do you reckon?

Reply:
Yeah, I would prefer e.g. "message handling loop".